.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     steps:
       - setup_remote_docker
       - checkout
-      - run: docker build . -t opinarygmbh/automation
+      - run: docker build . -t opinarygmbh/automation -t "opinarygmbh/automation:$CIRCLE_SHA1"
       - run: echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
       - run: docker push opinarygmbh/automation:latest
```

Comment:
Can we only push latest on master? This would cause some issues with the current set up

Reply:
would those issues be fixed if the runner image tag will be fixed to the hash in the automation cluster?

Reply:
Yes, if that's the approach then let's not build and push latest. Even though I would still prefer to have it built only on master